lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -10,7 +10,8 @@ define((require, exports, module) ->
             session.removeMarker(session.$bracketMismatchHighlight)
             session.$bracketMatchHighlight = null
             session.$bracketMismatchHighlight = null
-            toggleSurroundingBracketsPopup(editor)
+            if toggleSurroundingBracketsPopup
+                toggleSurroundingBracketsPopup(editor)
```

Comment:
@dbarashev 
I thought that it is possible to leave the possibility to use the API without this function.